Acquisition/Utilities/DataGenerator/source/dataGenerator.cpp

```diff
@@ -0,0 +1,72 @@
+///@file dataGenerator.cpp
+///@brief A program that will generate data when provided with a firmware and frequency.
+///@author S. V. Paulauskas
+///@date August 9, 2017
+///@copyright Copyright (c) 2017 S. V. Paulauskas.
+///@copyright All rights reserved. Released under the Creative Commons Attribution-ShareAlike 4.0 International License
+#include <array>
+#include <iostream>
+
+#include "XiaListModeDataEncoder.hpp"
+#include "hribf_buffers.h"
+
+using namespace std;
+
+int main(int argc, char *argv[]) {
```

Comment:
This is MVP correct? Otherwise it would be nice to take all these configuration variable as arguments.

Reply:
As mentioned, this is an MVP - minimum viable product. The MVP provides a base for further improvements, and constitutes a working piece of software. 

![image](https://user-images.githubusercontent.com/1553042/30545872-ebcdf504-9c58-11e7-9b35-77fb50bf6969.png)

Reply:
I think a floating face might be very happy with a single wheel.